### ISSUE TITLE
If on a winter's night a library cardholder

### ISSUE THREAD

**Issue by robincamille** [NaNoGenMo/2016#58]
Declaring my intention to participate in NaNoGenMo! Tentative project title: _If on a winter's night a library cardholder._

I've been meaning to take part in NaNoGenMo since 2013. This year's the year.

**Comment by ikarth** [comment]
Points for the name.

**Comment by robincamille** [comment]
Thanks! 

Repository: https://github.com/robincamille/nanogenmo2016 

Status: In progress. Rough draft done, just fiddly formatting things left to do, mostly.

**Comment by michelleful** [comment]
This was great! Using excerpts was a great idea and the ending was very satisfying. BTW (I didn't look at the code) -- the books weren't all red in real life, were they?

**Comment by robincamille** [comment]
No, that was just creative license  😃  Thanks for the kind remarks!

**Comment by robincamille** [comment]
**Status: [complete!](https://github.com/robincamille/nanogenmo2016)** 

I also had a hardback book printed from one of the novel's many outputs: 
![ifonawinters4](https://cloud.githubusercontent.com/assets/744272/20467639/89fe77be-af59-11e6-8869-beed98efd04d.jpg)

**Comment by greg-kennedy** [comment]
Reviewing past entries (it's that time again!) and ran across yours.  I love this.  Great use of the format and a lot of fun.
